code-challenges.js

```diff
@@ -4,65 +4,154 @@
 
 // Describing your process is required.
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in two strings and determines which of the two strings has more characters. Use the two sets of test variables provided.
 
-// Describe your process:
-
-// Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
-
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
-// Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Describe your process:
+ 
+// Create a function that determines whether a number is below, above, or at the boiling point of 212 degrees
+// Input: Any number
+//  If statement needed:
+//      If given number is less than 212 degrees, return text string "Below boiling point"
+//      If given number is greater than 212 degrees, return text string "Above boiling point"
+//      If given number is equal to 212 degrees, return text string "At boiling point"
+// Output: Below boiling point, above boiling point, at boiling point
 
-const temp1 = 42
+
+
+//const temp1 = 42
 // Expected output: "42 is below boiling point"
 
-const temp2 = 350
+//const temp2 = 350
 // Expected output: "350 is above boiling point"
 
-const temp3 = 212
+//const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+
+const boilingTemp = () => {
+    const temp1 = 42
+    const temp2 = 350
+    const temp3 = 212
+    if (temp1 > 212) {
+        return (temp1 + "is above boiling point")}
+    else if (temp1 < 212) {
+        return (temp1 + "is below boiling point")}
+    else if (temp1 === 212) {
+        return (temp1 + "is at boiling point")}
+
+    if (temp2 > 212) {
+        return (temp2 + "is above boiling point")}
+    else if (temp2 < 212) {
+        return (temp2 + "is below boiling point")}
+    else if (temp2 === 212) {
+        return (temp2 + "is at boiling point")}
+
+    if (temp3 > 212) {
+        return (temp3 + "is above boiling point")}
+    else if (temp3 < 212) {
+        return (temp3 + "is below boiling point")}
+    else if (temp3 === 212) {
+        return (temp3 + "is at boiling point")}
+    
+        console.log(boilingTemp(42))
+        console.log(boilingTemp(350))
+        console.log(boilingTemp(212))
+        }
+
+    // No results returned - Not sure why??
```

Comment:
Technically the function is never invoked here since the console.logs are inside the function currently.  An important thing to focus on that could help make this function more efficient would be parameters.  I would recommend reviewing parameters in the [Functions](https://github.com/learn-academy-2022-golf/Syllabus/blob/main/javascript/functions.md) section in the syllabus.